composer/utils/checkpoint.py

```diff
@@ -318,6 +318,11 @@ def load_sharded_checkpoint(
             f'Sharded checkpoint loading on >1 node requires torch version >= 2.0.1. You have torch version {torch.__version__}'
         )
 
+    if state.fsdp_config is None:
+        raise ValueError('Sharded checkpointing requires an FSDP config.')
```

Comment:
```suggestion
    if state.fsdp_config is None:
        raise ValueError('Loading a sharded checkpointing requires passing a FSDP config to Trainer.')
```
nit: more descriptive and grammar